components/ProductMatrix.vue

```diff
@@ -1,13 +1,24 @@
 <template>
   <div>
     <div ref="matrix" class="product-matrix">
-      <component
-        :is="components[card.type]"
-        v-for="card in cards"
-        :key="card.sortKey"
-        ref="card"
-        :card="card"
-      />
+      <template v-for="card in cards">
+        <div v-if="Array.isArray(card)" :key="card.sortKey" class="buddy">
+          <component
+            :is="components[_card.type]"
+            v-for="_card in card"
```

Comment:
Is there a better naming here?

Reply:
Any suggestions?

Reply:
Underscore seems to be used for "unused variables"!?